src/users/users.service.ts

```diff
@@ -116,7 +113,7 @@ export default class UsersService {
      */
     private async addAuthInterceptor() {
         this.axiosInstance.interceptors.request.use(async (config) => {
-            const token: string = await this.SpikeService.getToken();
+            const token = await this.SpikeService.getToken();
```

Comment:
why delete the type? 
Also, look for the type of the token. if there is none, put string. (SpikeService.getToken should return type Promise<string>)